extractor/dependencies.js

```diff
@@ -0,0 +1,41 @@
+const Ajv = require("ajv");
+const path = require("path");
+const fs = require("fs-extra");
+
+const getSchema = async function(options) {
+	if (options.dependenciesSchema)
+		return Promise.resolve(options.dependenciesSchema);
+
+	return fs.readJson(
+		path.join(options.schemaLocation, "schema.dependencies.json")
+	);
+};
+
+module.exports = async function(options) {
+	if (!options.dependencies) return Promise.resolve(null);
+
+	let getContent;
+	switch (typeof options.dependencies) {
+		case "string": {
+			getContent = fs.readJson(path.join(process.cwd(), options.dependencies));
```

Comment:
getDependencies should return null if the file doesn't exist.

Reply:
But it gives additional problems - the user expects the file to be included, and in fact there was an error in assigning the path to the file in the configuration

Reply:
fixed